models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md

```diff
@@ -0,0 +1,137 @@
+| mAP    | 35.9% |
+| [COCO\* mAP](http://cocodataset.org/#detection-eval) | 39.7% |
+
+## Input
+
+### Original model
+
+Image, name - `image_input`, shape - `1,416,416,3`, format is `B,H,W,C` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+Scale value - 255.
+
+### Converted model
+
+Image, name - `image_input`, shape - `1,3,416,416`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `conv2d_9/BiasAdd`,  shape - `1,13,13,255`. The anchor values are `81,82, 135,169, 344,319`.
+
+2. The array of detection summary info, name - `conv2d_12/BiasAdd`,  shape - `1,26,26,255`. The anchor values are `23,27, 37,58, 81,82`.
+
+For each case format is `B,Cx,Cy,N*85,`, where
+    - `B` - batch size
+    - `Cx`, `Cy` - cell index
+    - `N` - number of detection boxes for cell
+
+Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_no_80`], where:
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get relative to the cell coordinates
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get absolute height and width values
+- `box_score` - confidence of detection box, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get confidence in [0,1] range
+- `class_no_1`,...,`class_no_80` - probability distribution over the classes in logits format, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) and multiply by obtained confidence value to get confidence of each class
+
+### Converted model
+
+1. The array of detection summary info, name - `conv2d_9/BiasAdd/YoloRegion`,  shape - `1,255,13,13`. The anchor values are `81,82, 135,169, 344,319`.
+
+2. The array of detection summary info, name - `conv2d_12/BiasAdd/YoloRegion`,  shape - `1,255,26,26`. The anchor values are `23,27, 37,58, 81,82`.
+
+For each case format is `B,N*85,Cx,Cy`, where
+- `B` - batch size
+- `N` - number of detection boxes for cell
+- `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_no_80`], where:
+- (`x`,`y`) - coordinates of box center relative to the cell
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get absolute height and width values
+- `box_score` - confidence of detection box in [0,1] range
+- `class_no_1`,...,`class_no_80` - probability distribution over the classes in the [0,1] range, multiply by confidence value to get confidence of each class
+
+> **NOTE:** Original model uses [[3,4,5],[**1,2,3**]] (not [[3,4,5], [**0,1,2**]]) mask for anchors.
```

Comment:
not sure that it is right place for this comment, for me it will be better to specify anchor mask for each layer without emphasis that original repo has mistake